apps/console/src/features/core/configs/routes.tsx

```diff
@@ -71,6 +71,8 @@ import { AppConstants } from "../constants";
 
 export const getAppViewRoutes = (useExtendedRoutes: boolean = false): RouteInterface[] => {
 
+    const legacyFeatures: LegacyFeaturesInterface = window["AppUtils"].getConfig().ui.legacyFeatures;
```

Comment:
Shall we add null checks just in case :)

Reply:
Fixed with 8def5093ef2d045c6fddd9d597bbbe952ac756f3